relm4/src/component/async/traits.rs

```diff
@@ -38,6 +38,13 @@ pub trait AsyncComponent: Sized + 'static {
     /// Initializes the root widget
     fn init_root() -> Self::Root;
 
+    /// Allows you to initialize the root widget with a temporary value
+    /// as a placeholder until the [`init()`](AsyncComponent::init)
+    /// future completes.
+    ///
+    /// This method does nothing by default.
+    fn temporary_init(_root: &mut Self::Root) {}
```

Comment:
This is a cool idea! I don't really like the name though. "temporary" doesnt' really imply "loading" for me.

What do you think about `init_loading`? Or `init_placeholder`?

Reply:
I like your suggestions. Maybe `init_loading_widgets` would be even better?

Reply:
Works for me.

Reply:
Done